Add tokens records to certificate.

benoitdevos opened the pull request:
* Tokens records are shown on the certificate.
* The related files can be downloaded when user is authenticated.

logion-network/logion-internal#780

gdethier reviewed (approved):
Great! Just one remark: I think the asset is already there (`public/assets/records_polka.svg`).

benoitdevos commented:
> Great! Just one remark: I think the asset is already there (`public/assets/records_polka.svg`).

Yes, correct. I have removed it, and fixed some wordings, here: https://github.com/logion-network/logion-wallet/compare/3b937362ff1cb0af4f79e9e49698b2648f2c891c..6a4dcd121f16f097a88f0229e5a31536ff7ee3ab
